src/askvm/lib/run.ts

```diff
@@ -34,9 +39,9 @@ export async function run(
     return typed(code);
   }
 
-  ops += 1;
-  if (ops > 1000) {
-    // throw new Error('Over ops limit!');
+  options.stats!.steps += 1;
+  if (options.stats!.steps >= options.limits!.steps) {
```

Comment:
If think it should be > if we are saying we are *over* the limit.

Reply:
This is at the beginning of the `run` operation, so if we are at the limit, then we cannot proceed.